src/main/java/org/kainos/ea/validator/JobSpecValidator.java

```diff
@@ -1,6 +1,4 @@
 package org.kainos.ea.validator;
 
 public class JobSpecValidator {
```

Comment:
Do we need this to view a list of job roles?

Reply:
No, it's for other user stories

Reply:
I've deleted this as we're discussing changes